jobs/future-effective-filings/openshift/cron-file-future-effective.yml

```diff
@@ -0,0 +1,70 @@
+---
```

Comment:
All of our other templates are in json, please re-export as json.

Reply:
These are yaml for the other jobs?

Reply:
Ugh, seems some got through as YAML. Leave it for now then, and I'll change them in another ticket.